06week/Checkpoint2.js

```diff
@@ -0,0 +1,45 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = strNums.join();
+
```

Comment:
Nums should be an array of numbers not strings. So, for every number in strNums, convert to a number, push to new array. Map would be best. 
const nums = strNums.map(str => {
    return Number(str)
})